src/indexing_sources/source_pgsql.cpp

```diff
@@ -111,6 +115,8 @@ struct CSphSource_PgSQL : CSphSource_SQL
 	DWORD			SqlColumnLength ( int iIndex ) final;
 	const char *	SqlColumn ( int iIndex ) final;
 	const char *	SqlFieldName ( int iIndex ) final;
+	const char *	SqlColumnStream ( int iIndex, DWORD &iStreamLength ) final;
+	void			SqlColumnFreeStream (char *stream ) final;
```

Comment:
char* should be prefixed with 's'. I.e - `sStream` here.
If it is asciiz - better is even 'sz' - i.e. `szStream`.